tests/baselines/reference/privateNameFieldAssignment.js

```diff
@@ -52,8 +52,8 @@ var __classPrivateFieldGet = (this && this.__classPrivateFieldGet) || function (
 var _field;
 class A {
     constructor() {
-        var _a, _b, _c, _d, _e, _f, _g, _h, _j, _k, _l, _m;
         _field.set(this, 0);
+        var _a, _b, _c, _d, _e, _f, _g, _h, _j, _k, _l, _m;

```

Comment:
I bet it's from the prologue ordering being more manual (https://github.com/microsoft/TypeScript/pull/29374#discussion_r533913684). Do you see it as an issue? I don't see a case where this would cause a real behavioral difference in the emitted code.